lib/unread/readable.rb

```diff
@@ -124,9 +125,14 @@ def mark_as_read!(options)
       end
 
       def read_mark(reader)
+        assert_reader(reader)
         read_marks.where(:reader_id => reader.id, reader_type: reader.class.base_class.name).first
       end
 
+      def read_at(reader)
+        unread?(reader) ? nil : read_mark(reader).read_at
```

Comment:
I'm not a fan of returning `nil` here instead of a DateTime object, but I think checking `.nil?` is more obvious than checking for some ominous `THIS_IS_ACTUALLY_UNREAD_TIMESTAMP`.